src/content/en/tools/chrome-devtools/issues-tab/index.md

```diff
@@ -0,0 +1,134 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Use the Issues Tab to find and fix problems with your website.
+
+{# wf_updated_on: 2020-05-13 #}
+{# wf_published_on: 2020-05-16 #}
+{# wf_blink_components: Security #}
+
+# Find and Fix Problems With the Chrome DevTools Issues Tab {: .page-title }
```

Comment:
```suggestion
# Find And Fix Problems With The Chrome DevTools Issues Tab {: .page-title }
```
In the DevTools docs I for some reason decided to capitalize every word in the title. I don't know if that's a good convention anymore but this will at least keep it consistent with the rest of the docs.